src/graphql/data/util.tsx

```diff
@@ -133,7 +133,9 @@ export const CHAIN_NAME_TO_CHAIN_ID: { [key in Chain]: SupportedChainId } = {
 }
 
 export function fromGraphQLChain(chain: Chain): SupportedChainId {
-  return CHAIN_NAME_TO_CHAIN_ID[chain]
+  const chainId = CHAIN_NAME_TO_CHAIN_ID[chain]
+  if (!chainId) throw new Error(`Unknown chain: ${chain}`)
```

Comment:
we don't want to error in this case as on unsupported chains we show Mainnet data